routes/admin/profile.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var router = express.Router();
+var session = require('express-session');
+
+var adminsModel = require('../../schemas/admins');
+var portalsModel = require('../../schemas/portals');
+var settingsModel = require('../../schemas/settings');
+var coursesModel = require('../../schemas/courses');
+
+
```

Comment:
`adminsModel`, `portalsModel`, `settingsModel`, `session` are not needed in this file.